GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1979,6 +1975,13 @@ public class ChangeCount
             public IReadOnlyList<GitItemStatus> Deleted { get; set; }
             public IReadOnlyList<GitItemStatus> SubmodulesChanged { get; set; }
             public IReadOnlyList<GitItemStatus> SubmodulesDirty { get; set; }
+
+            public bool Any
```

Comment:
This is a poor name because it looks very similar to `Linq.Any()`.
```suggestion
            public bool HasChanges
```